### ISSUE TITLE
Ошибка при запуске mvn clean test

### ISSUE THREAD

**Issue by breakmt** [allure-framework/allure-core#545]
Пытаюсь запустить пример:
https://github.com/allure-examples/allure-testng-example

Выполнил:
````
git clone git@github.com:allure-examples/allure-testng-example.git
````

Дальше делаю:
````
mvn clean test
````

Но в итоге ошибка:
````
d:\demo\allure-testng-example>mvn clean test
[INFO] Scanning for projects...
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]
[ERROR]   The project ru.yandex.qatools.allure:allure-testng-example:1.0-SNAPSHOT (d:\demo\allure-testng-example\pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM: Could not find artifact ru.yandex.qatools.allure:allure-examples-parent:pom:1.0-SNAPSHOT and 'parent.relativePath' points at wrong local POM @ line 7, column 13 -> [Help 2]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/UnresolvableModelException
````

Apache Maven 3.2.5

**Comment by baev** [comment]
Add Sonatype snapshot repository to your `~/.m2/settings.xml`

```xml
<repository>
    <id>snapshots-repo</id>
    <url>https://oss.sonatype.org/content/repositories/snapshots</url>
    <releases><enabled>false</enabled></releases>
    <snapshots><enabled>true</enabled></snapshots>
</repository>
```

**Comment by baev** [comment]
btw I released `1.0` version of `allure-examples-parent`, all should work fine within 2 hours.

**Comment by baev** [comment]
@breakmt could you please update example (`git pull origin master`). All should work now.
